8.x/theme-development/representative.md

```diff
@@ -0,0 +1,381 @@
+
+```php
+  @include("_register-representative") #line 65
+```
+The included file contains a form field for entering representative information during the registration process using `representative_is_enabled()` helper function.
+
+```php{3}
+// public/views/register-representative.blade.php
+
+@if(representative_is_enabled())
+    <div class="col-md-6 col-md-6 col-xs-12">
+        <div class="form-group">
+            <label for="representative_type">نحوه آشنایی با مجموعه</label>
+            <select class="form-control number-control" name="representative_type">
+                <option value="">هیج کدام</option>
+                <option value="مشتریان فعلی مجموعه" id="manual-representative">مشتریان فعلی مجموعه</option>
+                @foreach(representative_get_options() as $option)
+                    <option value="{{$option}}">{{$option}}</option>
+                @endforeach
+            </select>
+        </div>
+    </div>
+    </div>
+    <div class="col-md-6 col-sm-6 col-xs-12">
+        <div class="form-gruop">
+            <label for="family">شماره تماس معرف در صورت تمایل</label>
+            <input type="text" class="form-control number-control" name="representative_username"
+                placeholder="شماره تماس معرف"
+                value="{{ old('representative_username') }}">
+        </div>
+    </div>
+@endif
+```
+
+So now if you activate the representative form from admin pannel, and double check the registration form, the representative sections will be disappeared.
+
+![section is enable](/repIsenable.png)
+
+### How to pass list options 
+
+#### representative_get_options()
+
+Another helper function used in this form is `representative_get_options`. This fnction displays the list of user selectable options with the help of a `@foreach` loop. Here the resulting list is then displayed on the registeration form as a drop-down menu, allowing the user to indicate how they become acquainted with you.
+
+```php{6-8}
+// public/views/register-representative.blade.php
+
+<select class="form-control number-control" name="representative_type">
+     <option value="">هیج کدام</option>
+    <option value="مشتریان فعلی مجموعه" id="manual-representative">مشتریان فعلی مجموعه</option>
+         @foreach(representative_get_options() as $option)
+            <option value="{{$option}}">{{$option}}</option>
+        @endforeach
+</select>
+
+```
+
+You can replace a radio button instead of a select/option tag.
+
+```php
+
+@foreach(representative_get_options() as $option)
+<lable for="">
+    {{$option}}
+<input type="radio" class="form-control" name="representative_type"
+  value="{{$option}}">
+ </lable>           
+@endforeach
+
+```
+
+Now deploy the project and see the result :
+```
+./deploy.sh
+```
+![Drop-down](/repDropDown.png)
+
+**Note 1:** Options were already passed in the form manually.
+
+**Note 2:** Consider not changing the "name" attribute.
+
+**Note 3:** You can add CSS to adjust options location on the form.
```

Comment:
You can add some css styles to adjust the elements position on the registration form.